elfeed-org.el

```diff
@@ -36,6 +36,7 @@
 (require 'org)
 (require 'dash)
 (require 's)
+(require 'rx)
```

Comment:
This library also needs to be added to ``;; Package-Requires : ((elfeed "1.1.1") (org "8.2.7") (dash "2.10.0") (s "1.9.0"))``` for it to be installed correctly.